proto/substrait/validator/validator.proto

```diff
@@ -4,6 +4,7 @@ syntax = "proto3";
 package substrait.validator;
```

Comment:
It might make sense to add `proto/** linguist-generated=true` to `.gitattributes` so that generated code isn't expanded in review.

Reply:
That's a really cool feature I didn't know about! However, those protos are not generated. IIRC the only generated file that's checked in is the Substrait version file, because it currently doesn't generate correctly in all circumstances (lack of history information, failure to call git, etc).